cfgov/unprocessed/css/enhancements/forms.less

```diff
@@ -376,22 +224,7 @@ input {
     }
 }
 
-/* topdoc
-  name: Ensure form legends wrap in IE
-  family: cf-form
-  notes:
-    - "Legends do not wrap in IE. Different styles are
-      required to ensure wrapping in different versions."
-  tags:
-    - cf-form
-*/
-
-legend {
```

Comment:
This one too, I don't see it in CF.

Reply:
https://github.com/cfpb/capital-framework/blob/canary/src/cf-forms/src/atoms/legend.less

Reply:
Okay if markup is updated 👍

Reply:
The only one I'm not sure about [is this](https://github.com/cfpb/cfgov-refresh/blob/9964c5cb3a4bfcee228f0184a1a0032f92f343b9/cfgov/jinja2/v1/_includes/conference-registration-form.html#L90). Is this still in use?

Reply:
As far as I know yes, annually. There's an option in Wagtail to add the conference registration form. @chosak knows.